d2cli/main.go

```diff
@@ -88,6 +89,11 @@ func Run(ctx context.Context, ms *xmain.State) (err error) {
 	if err != nil {
 		return err
 	}
+	timeoutFlag, err := ms.Opts.Int64("D2_TIMEOUT", "timeout", "", 0, "the number of seconds before d2 will timeout. (default=120)")
```

Comment:
add to man page

Reply:
you can't pass in `0` as default and say 120. the flag library will print default 0.